release-notes.md

```diff
@@ -0,0 +1,45 @@
+# FireCloud Explorer Release Notes
+
+### Alpha version 0.1.0
+
+## Feature summary
+
+This first alpha release of FireCloud Explorer includes the following features:
+
+● Login using FireCloud credentials.
```

Comment:
If these are list elements I think you need to use `*` (using `+` or `-` also works)

Reply:
actually, * is a different type of bullet, but I can update it